webcompat/api/uploads.py

```diff
@@ -86,6 +86,8 @@ def save(self):
         # Optimize further the image compression for these formats
         if self.image_object.format in ['JPEG', 'JPG', 'JPE', 'PNG']:
```

Comment:
the save_parameters['optimize'] is useful in the case of JPEG and doesn't have a strong impact on performance. If we exclude the 'PNG', we will need to add it for making sure we are saving to JPG.